src/main/java/hudson/plugins/emailext/ExtendedEmailPublisherDescriptor.java

```diff
@@ -54,6 +54,7 @@
  * These settings are global configurations
  */
 @Extension
+@Symbol("email-ext")
```

Comment:
This is not backward-compatible unfortunately.

Reply:
I would suggest

```java
@Symbol({"email-ext", "extendedEmailPublisher"})
```

if CasC in fact behaves properly: accepting either, but suggesting only the first. @timja @jetersen et al.?

Reply:
Yes casc works with that, anyone using the previous behaviour will also prompt an administrative warning telling them to update to the new syntax:

e.g. (on my instance which I haven't got around to fixing)

![image](https://user-images.githubusercontent.com/21194782/103560495-c5d13e80-4eaf-11eb-9b17-979f58629b44.png)

Reply:
I'm unclear whether this is actually breaking compatibility, or not. IOW, is this supposed to be only a warning, or an instance using the previous values would not start without adapting their yaml (or wait for https://github.com/jenkinsci/email-ext-plugin/pull/271)

Reply:
it depends on whether someone was using the previous symbol for something or whether any jcasc was possible before without this